src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/dto/CreateLegacyContactOnVisitRequestDto.kt

```diff
@@ -6,7 +6,7 @@ import javax.validation.constraints.NotBlank
 
 data class CreateLegacyContactOnVisitRequestDto(@field:NotBlank val name: String, @field:NotBlank val telephone: String) {
   companion object {
-    private const val UNKNOWN = "UNKNOWN"
+    const val UNKNOWN = "Unknown"
```

Comment:
I'm not sure if this should be done, on server especially the entity because we would have migrated the data already! you could certainly change it in the DTO being returned. But I'm not rally sure if this is a concern of the entity or even the server side, it is a display concern.

Reply:
If you do this, you would have to update the entries in the DB!

Reply:
There is defiantly an inconstancy in how the UI is representing this data - im not sure the UI should be manipulating this text.  The api should be omitting unknown fields from the response, but these fields are NOT NULL & NOT BLANK for VSiP so we decided to populate migrated records with an unknown text value.  I believe the UI should display the text as it receives it in this case, otherwise were have code in the UI to handle a temporary fix for migrated records.

With the existing migrated data, we only have dev and test migration data at present which we can drop and re-migrate or leave and migrate some new records for testing this change.  As far as im aware nothing has been migrated yet in the PROD environment.

Reply:
Ignoring the suggested fix on the defect.  Another solution to this issue is that the UI simply displays the text as is which would be all uppercase for unknown contact data.  The assumption in the defect from planning is that the capitalisation of the contact name is correct and the phone number is wrong but we can easily say the reverse which means leaving the API as is and updating the UI to remove the manipulation of the contact name when the name is unknown - this would still need a change in the UI anyway.

Reply:
I think the last suggestion is correct.. it's a UI change as it is a UI concern as far as the server side is concerned it's fine... we should not have inconsistency in the DB